posthog/async_migrations/utils.py

```diff
@@ -35,11 +35,17 @@ def execute_op_postgres(sql: str, query_id: str):
         cursor.execute(f"/* {query_id} */ " + sql)
 
 
-def process_error(migration_instance: AsyncMigration, error: str, rollback: bool = True, alert: bool = False):
+def process_error(
+    migration_instance: AsyncMigration,
+    error: str,
+    rollback: bool = True,
+    alert: bool = False,
+    status: int = MigrationStatus.Errored,
```

Comment:
use the MigrationsStatus type

Reply:
It can't be a type - `MigrationStatus.Errored` (int) is not of the same type as `MigrationStatus` (class). This operates nicely as an enum but can't use as a type in that way.